lib/runner/extensions/request.command.js

```diff
@@ -35,6 +35,9 @@ var _ = require('lodash'),
         // get a reference to the Auth instance from the original item, so any changes don't have to be synced back
         context.auth = payload.item.getAuth();
 
+        // keep track of no of replays of the request
+        context.replayCount = context.replayCount || 0;
```

Comment:
We should have a better name for this property IMO. It's not actually the "replay" count, it's the number of times the request went through the entire pipeline. @shamasis ?